src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -22,33 +22,28 @@
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
-
-    public static final Times EMPTY_TIMES = new Times("");
-
     public static Book[] getSampleBooks() {
         HashMap<String, Integer> storage = new HashMap<>();
         storage.put("centralLibrary", 10);
-        storage.put("scienceLibrary", 10);
+        storage.put("scienceLibrary", 8);
```

Comment:
Thanks for changing the number to a different one from the stocking number of the other library.